tendrl/node_agent/monitoring/collectd/collectors/gluster/heavy_weight/tendrl_gluster_heal_info.py

```diff
@@ -61,7 +62,8 @@ def get_volume_heal_info_split_brain_stats(vol, integration_id, etcd_client):
     except (
         AttributeError,
         KeyError,
-        ValueError
+        ValueError,
+        ElementTree.ParseError
```

Comment:
I saw some unhandled exceptions, so i added ElementTree.ParseError

Reply:
In what scenario this exception can happen? I feel you should add some comment as well to explain that.

Reply:
I will comment, This will happen when heal info command gives None. During gluster heal info command  timeout or lower version of gluster without nolog command support this error will come